gen_cert.py

```diff
@@ -149,6 +151,7 @@ def interstitial_factory():
                 for key, value in cert_data.get('interstitial', {}).items()
             }
             self.interstitial_texts.update(interstitial_dict)
+            self.timezone = cert_data.get('timezone', settings.DEFAULT_TIMEZONE)
```

Comment:
More form over function, but I wonder if we can drop the `DEFAULT_` prefix?

I can't imagine how we would ever need both `settings.DEFAULT_TIMEZONE` _and_ `settings.TIMEZONE`, so it feels like a superfluous specification. That we're pulling it directly from `settings` already implies that this is a system-level setting. I guess it just feels like extra letters that add no additional clarity.

So on a high level, this line of code reads like "try to get the course-specific timezone, but coalesce to the system-settings-specific timezone".